docs/api/v2/node.md

```diff
@@ -8,20 +8,54 @@ TBD.
 
 *NOTE: Some of these are currently at the protocol layer rather than than the Node itself.*
 
-```
+### WakuSub API
+
+```Nim
 method publish*(w: WakuSub, topic: string, data: seq[byte]) {.async.}
 method subscribe*(w: WakuSub, topic: string, handler: TopicHandler) {.async.}
+method unsubscribe*(w: WakuSub, topics: seq[TopicPair]) {.async.}
+```
+
+### Waku API
+
+```Nim
+proc publish(w: WakuNode, topic: WakuTopic, data: seq[byte]): bool
+proc subscribe(w: WakuNode, topics: seq[string], handler: WakuHandler): Identifier
+proc unsubscribe(id: Identifier): bool
+# Can get rid of the identifiers here and do something like WakuSub.
+
+# TODO: How can these work at all? We can't be selective on this at the level
```

Comment:
Agree, we need to extend https://github.com/vacp2p/specs/issues/156 here as you mentioned

Reply:
I'd leave this as a TODO for the purposes of this issue

Reply:
ok, I'll document this a bit better in comments then.